src/mon/PGMap.h

```diff
@@ -49,7 +49,7 @@ class PGMapDigest {
   pool_stat_t pg_sum;
   osd_stat_t osd_sum;
   mempool::pgmap::map<std::string,osd_stat_t> osd_sum_by_class;
-  mempool::pgmap::unordered_map<uint64_t,int32_t> num_pg_by_state;
+  mempool::pgmap::unordered_map<uint64_t,int64_t> num_pg_by_state;
```

Comment:
no, i think the original version is correct. please see `PGMap::stat_pg_add()`, where we have
```c++
num_pg_by_state[s.state]++;
```

so the key of the map is state, and the value is the number of pg with that state.